cfgov/permissions_viewer/templates/permissions_viewer/group.html

```diff
@@ -0,0 +1,28 @@
+{% extends "wagtailadmin/base.html" %}
+{% load i18n %}
+{% load gravatar %}
+{% block titletag %}{% trans "Roster and Permissions for" %} {{group}}{% endblock %}
+
+{% block content %}
+    {% include "wagtailadmin/shared/header.html" with title="Members and Permissions" %}
+
+    <div class="nice-padding">
+	    <h1> {{group}} </h1>
+	    <h2>Active Users</h2>
+	    <ul>
+	{% for user in active_users %}
+	<li><a href="/admin/permissions/user/{{user.pk}}/">{{ user }}</a></li>
+	{% empty %}
+	<li>There are no active users</li>
+	{% endfor %}
+	 </ul>
+	    <h2>Inactive Users</h2>
+	    <ul>
+	{% for user in inactive_users %}
+	<li><a href="/admin/permissions/user/{{user.pk}}/">{{ user }}</a></li>
```

Comment:
Can this be changed to use a `{% url %}` tag that reverses `permissions:user`?

Reply:
done!